datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -55,8 +55,8 @@ def __init__(self):
     def is_aggregate(cls, mtype):
         return mtype in cls.AGGREGATE_TYPES
 
-    def submit_metric(self, check, check_id, mtype, name, value, tags, hostname):
-        self._metrics[name].append(MetricStub(name, mtype, value, tags, hostname))
+    def submit_metric(self, check, check_id, mtype, name, value, tags, hostname, device=None):
```

Comment:
added default `device=None` to avoid having to add it on base check

Reply:
I don't think you can change this signature, it's the point of the revert, no?

Reply:
I cannot change it on the base check, this is the stub

Reply:
Doesn't feel quite right to have a mismatch between the stub signature and the real one.

Another option is just avoiding asserting the `device` in e2e ? Seems enough to assert it in integration.

Reply:
Yes, that's another option